ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2068,11 +2064,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 301;
-				DEVELOPMENT_TEAM = J22M3M9VYU;
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = J22M3M9VYU;
+				DEVELOPMENT_TEAM = 578K4HCZ2Y;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code signing style has been changed from Manual to Automatic, and the development team has been updated. If these changes were intentional and the new settings are correct, then it's fine. However, if these changes were not intended, they could potentially lead to issues with app signing and distribution.

```diff
- "CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
- CODE_SIGN_STYLE = Manual;
- DEVELOPMENT_TEAM = J22M3M9VYU;
- "DEVELOPMENT_TEAM[sdk=iphoneos*]" = J22M3M9VYU;
+ CODE_SIGN_STYLE = Automatic;
+ DEVELOPMENT_TEAM = 578K4HCZ2Y;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->